### ISSUE TITLE
Who cares if atmos devices are powered or unpowered, you can bypass APCs apparently

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#17220]
#### Description of issue
You can toggle off APC environment power, unwrench and rewrench "powered" devices (pumps), and they will function with no power usage.  On that note, "unpowered" devices like passive gates can not be interfaced with when environment power is off, regardless of being labeled an unpowered module.

#### Difference between expected and actual behavior
For powered devices to require power and for unpowered devices to not require power.

#### Steps to reproduce 
Turn off environment power of the APC, either make a new "powered" device or just unwrench and rewrench a current one.

**Comment by PJB3005** [comment]
Pretty sure this is the same bug allowing things like pumps to be powered in space, and fixing that one would piss a whole lot of people off so...

**Comment by ghost** [comment]
![y5rd65g65sd](https://user-images.githubusercontent.com/20872793/35359000-31a2960a-011e-11e8-801d-96ad674e07d1.jpg)

**Comment by Aurx** [comment]
Clearly the answer is to rework atmos machinery to draw power from a wire under knot under it instead of from the APC.  
This will not cause any issues or unnecessary headaches for mappers and players.

**Comment by FudgePucker** [comment]
The fury of 100000000 *aflaps will be unleashed if you make atmos devices require APCs.

**Comment by StreptoMutans** [comment]
Probably related to the bug which allows you to power computers by un- and rescrewing the monitor

**Comment by ghost** [comment]
Yeah, but who knows what to do cause you can't exactly colonize space with the types of rounds we have. Just pretend it's power over wi-fi
